doc/api/diagnostics_channel.md

```diff
@@ -307,6 +307,79 @@ channel.subscribe(onMessage);
 channel.unsubscribe(onMessage);
 ```
 
+### Built-in Channels
+
+* `http.client.request.start`: Emited with arguments `request` when client
+  starts a request.
+* `http.client.response.finish`: Emited with arguments `request`, `response`
+  when client receives a response.
+* `http.server.request.start`: Emited with arguments `request`, `response`,
+  `socket`, `server` when server receives a request.
+* `http.server.response.finish`: Emited with arguments `request`, `response`,
+  `socket`, `server` when server sends a response.
+
+## Example
+
+```mjs
+import http from 'node:http';
+import dc from 'node:diagnostics_channel';
+
+const onClientRequestStart = dc.channel('http.client.request.start');
+const onClientResponseFinish = dc.channel('http.client.response.finish');
+
+onClientRequestStart.subscribe(({ request }) => {
+
+});
+
+onClientResponseFinish.subscribe(({ request, response }) => {
+
+});
+
+const server = http.createServer((req, res) => {
+  res.end('done');
+});
+
+server.listen(() => {
+  const { port } = server.address();
+  http.get(`http://localhost:${port}`, (res) => {
+    res.resume();
+    res.on('end', () => {
+      server.close();
+    });
+  });
+});
+```
+
+```cjs
+const http = require('http');
+const dc = require('diagnostics_channel');
+
+const onClientRequestStart = dc.channel('http.client.request.start');
+const onClientResponseFinish = dc.channel('http.client.response.finish');
+
+onClientRequestStart.subscribe(({ request }) => {
+
+});
+
+onClientResponseFinish.subscribe(({ request, response }) => {
+
+});
+
+const server = http.createServer((req, res) => {
+  res.end('done');
+});
+
+server.listen(() => {
+  const { port } = server.address();
+  http.get(`http://localhost:${port}`, (res) => {
+    res.resume();
+    res.on('end', () => {
+      server.close();
+    });
+  });
+});
+```
+
```

Comment:
The Overview section already provides a similar but more generic example. I'm not convinced we need this extra example section. I feel like a more specific example like this would be more suitable for guides, if you'd be interested in putting something there. 🤔

Reply:
> The Overview section already provides a similar but more generic example. I'm not convinced we need this extra example section. I feel like a more specific example like this would be more suitable for guides, if you'd be interested in putting something there. 🤔

What do the `guide` means ?  Can you provide a url ?

Reply:
See:
- https://nodejs.dev/learn
- https://github.com/nodejs/nodejs.dev

Reply:
> See:
> 
> * https://nodejs.dev/learn
> * https://github.com/nodejs/nodejs.dev

Thanks ! Is there anything else in the docs(diagnostics_channel.md) that needs to be updated ?